js/components/user/User.js

```diff
@@ -251,22 +259,26 @@ class User extends React.Component {
                     <div className="buttons-line-height mt-3 text-center">
                         {this._impersonateButton()}
                         {this._passwordChange()}
-                        {this._saveAndSendEmailButton()}
-                        {(currentUser.role === ROLE.ADMIN || currentUser.username === user.username) &&
-                        <Button variant='success' size='sm' ref='submit' className="d-inline-flex"
-                                disabled={!UserValidator.isValid(user) || userSaved.status === ACTION_STATUS.PENDING}
-                                onClick={() => this._onSave()}
-                                title={this.i18n('required')}>
-                            {this.i18n('save')}
-                            {!UserValidator.isValid(user) &&
-                            <HelpIcon className="align-self-center" text={this.i18n('required')}/>}
-                            {userSaved.status === ACTION_STATUS.PENDING &&
-                            <LoaderSmall/>}
-                        </Button>
+                        {!isUsingOidcAuth() &&
```

Comment:
Save button should be allowed because we need to change the organization

Reply:
Resolved with c0fe1368f5cb11269b3f56863ba910e878294afc